code/datums/uplink_item.dm

```diff
@@ -1032,7 +1014,7 @@ var/list/uplink_items = list()
 	This will not work when the singularity is still in containment. A singularity beacon can cause catastrophic damage to a space station, \
 	leading to an emergency evacuation. Because of its size, it cannot be carried. Ordering this sends you a small beacon that will teleport the larger beacon to your location upon activation."
 	item = /obj/item/device/sbeacondrop
-	cost = 14
+	cost = 10
```

Comment:
Nah, 14 is a good price for this. I buy this from time to time as station engineer and cut the emitter wires and let it cut a path right through the station. It's seriously good and the steep price is worth it imo.